Jenkinsfile

```diff
@@ -141,20 +135,15 @@ EOF
 										set -o pipefail
 										npm run cypress:run |tee cypress.log
 										ret=$?
-										if [ $ret -eq 0 ]; then
-										  grep --extended-regexp --only-matching 'https://dashboard.cypress.io/#/projects/1it63b/runs/[0-9]+' cypress.log |tail --lines=1 >.cypress_url
-										  echo $ret >.cypress_status
-										else
+										if [ $ret -ne 0 ]; then
 										  FAILED_TESTS="$( awk '/Spec/{f=1}f' cypress.log |grep --only-matching '✖ .*.feature' |awk '{ print "test/cypress/features/"$2 }' |xargs| tr -s ' ' ',' )"
-										  cd $WORKSPACE/$BRANCH_NAME
-										  make coldstart
-										  export CYPRESS_coreUrl=http://127.0.0.1:$( docker-compose port lisk 4000 |cut -d ":" -f 2 )
-										  sleep 10
-										  cd -
-										  npm run cypress:run -- --record --spec $FAILED_TESTS |tee cypress.log
-										  ret=$?
-										  grep --extended-regexp --only-matching 'https://dashboard.cypress.io/#/projects/1it63b/runs/[0-9]+' cypress.log |tail --lines=1 >.cypress_url
```

Comment:
https://github.com/LiskHQ/lisk-desktop/issues/2645 only mentions removing the github notification but not the "2 stages" run

Reply:
I moved it into the changed if instruction `if [ $ret -ne 0 ]; then` so it's there but in then instead of else